lib/perl/Genome/Site/TGI/Extension/generate_lims_disk_unarchive.pl

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env lims-perl
+
+use strict;
+use warnings;
+
+use Data::Dumper;
+use Carp qw/confess/;
+
+use GSCApp;
+
```

Comment:
Empirically no, since we've been using this successfully so far. :smile:  It's a good idea, though, so I've added it!